src/api/nodeRpcCall.js

```diff
@@ -183,7 +183,7 @@ const getLldBalances = async (addresses) => {
   }
 };
 
-const getAdditionalAssets = async (address) => {
+const getAdditionalAssets = async (address, isIndexNeed = false) => {
```

Comment:
I dont think this is required ? Some merge issue maybe ?

Reply:
isIndexNeed is not required that's why i put there default value, 

there is one usage in getAdditionalAssetsWorker where it's used

`function* getAdditionalAssetsWorker() {
  const walletAddress = yield select(blockchainSelectors.userWalletAddressSelector);
  const additionalAssets = yield call(getAdditionalAssets, walletAddress);
  yield put(walletActions.getAdditionalAssets.success(additionalAssets));
}`

I know i can leave it undefined, but i want to keep type, because i worked with typescript and i think, that this is a good practice.

Reply:
Okay but why is this part of "FE referendum 'proposal by' field should show identity" ?
I m not against merging small improvements like this, but you need to make it clear if its an unrelated small improvement or somehow part of the ticket

Reply:
Okay, you mean, is this function required here, not parameter. So this is on develop right now, this function is changed to other task, so merge issue.